### PR TITLE
add MakeDataTransferObjectFactoryCommand & reset folder structure

### PR DESCRIPTION
**Pull request by regnerisch**

### CODE REVIEW

**Comment by regnerisch** [comment]
Resolving #56 and #58

File: src/Commands/MakeDataTransferObjectFactoryCommand.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Commands;
+
+use Regnerisch\LaravelBeyond\Resolvers\AppNameSchemaResolver;
+use Regnerisch\LaravelBeyond\Resolvers\DomainNameSchemaResolver;
+
+class MakeDataTransferObjectFactoryCommand extends BaseCommand
```
Comment (alexgaal) [inline src/Commands/MakeDataTransferObjectFactoryCommand.php:8]: Maybe we should think about creating a DTO and a DTO factory by one command? Or at least provide a flag here for creating a DTO.

File: src/LaravelBeyondServiceProvider.php [hunk 2]
```diff
@@ -40,6 +41,7 @@ public function boot()
                 MakeCommandCommand::class,
                 MakeControllerCommand::class,
                 MakeDataTransferObjectCommand::class,
+                MakeDataTransferObjectFactoryCommand::class,
```
Comment (alexgaal) [inline src/LaravelBeyondServiceProvider.php:44]: I'd like to refactor this. I'll provide a change here.

File: src/Commands/MakeDataTransferObjectFactoryCommand.php (hunk 1 above)
Comment (regnerisch) [reply src/Commands/MakeDataTransferObjectFactoryCommand.php:8]: Feel free to add the required code :)

Comment (alexgaal) [reply src/Commands/MakeDataTransferObjectFactoryCommand.php:8]: I'll provide a separate PR for this feature.

**Review by alexgaal — APPROVED** [review]

**Review by alexgaal — APPROVED** [review]

**Pull request merged**